### PR TITLE
Sensors page rework

### PR DESCRIPTION
**Pull request by mhauru**
* Change the columns in the sensor list to use the new IDs
* Make the Edit button in the sensor list point to a functioning page where one can edit the attributes of a sensor, and add new locations/installations.

Closes https://github.com/alan-turing-institute/CROP/issues/224
Partially addresses https://github.com/alan-turing-institute/CROP/issues/215

### CODE REVIEW

File: __app__/crop/structure.py [hunk 1]
```diff
@@ -299,6 +299,9 @@ class SensorClass(BASE):
         nullable=False,
     )
     device_id = Column(Unicode(100), nullable=False)
+    aranet_code = Column(Unicode(64), nullable=True)
```
[the remark this answers is not in the document]
Comment (nbarlowATI) [reply __app__/crop/structure.py]: hmm... weird, they do seem to be there ok in `develop`.   No harm though I guess, as long as they do end up in the code after all the merging is done :)

**Comment by mhauru** [comment]
If you want to test the page, sensor number 50 on `cropapptest` is a dummy one I made for that purpose.

**Comment by mhauru** [comment]
The pages that consume sensor locations aren't ready to handle sensors with location history though. See comments in https://github.com/alan-turing-institute/CROP/issues/215

**Comment by nbarlowATI** [comment]
Very nice! :)
I think we can maybe do some UX testing on Thursday though - even though I knew I was looking for an "Edit" button, it took me quite a while to find it at the end of each row in the table (though on the other hand it is neat that way, in that the user doesn't need to specify which sensor they want to edit).   Maybe we can also have a prominent "Edit/Move a sensor" button, if other users also take a while to find it?

Also, in terms of what columns we show - it could be good to also show the "aranet_pro_id"?  We are running short of space, but potentially we could condense Aisle/Column/Shelf into a single "Location" column - what do you think?

**Comment by mhauru** [comment]
I like both ideas, will implement.

**Comment by mhauru** [comment]
Done. @nbarlowATI wanna take another look? There's a new navigation item in the sidebar, for editing sensors. The columns in the sensor list have also changed.

**Pull request merged**